tests/e2e/util.go

```diff
@@ -2743,6 +2743,26 @@ func deleteFcdWithRetriesForSpecificErr(ctx context.Context, fcdID string,
 	return waitErr
 }
 
+//updateDeploymentImage updates the image in the deployment with the given image
+func updateDeploymentImage(ns string, image string, deploymentName string, imageName string) {
```

Comment:
I don't see it being used anywhere else in the PR.

Reply:
yes this method is not required now, I removed it.